src/main/java/com/google/devtools/build/lib/actions/PathMapper.java

```diff
@@ -105,6 +105,16 @@ default String getMappedExecPathString(ActionInput artifact) {
     return map(artifact.getExecPath()).getPathString();
   }
 
+  /**
+   * Returns the length of the mapped path minus the length of the unmapped path.
```

Comment:
I got confused - this is actually supposed to return a negative number when the mapped path is shorter (what we expect). I think it makes more sense to deal with positive numbers. What do you think about making this return `|unmapped| - |mapped|` instead? Either way, please make it super clear in the documentation.